tests/autotune_test_yamls/manifests/da/app_autotune_yaml/datasource_name/valid-datasource-name.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: "recommender.com/v1"
+kind: "Autotune"
+metadata:
+ name: "valid-datasource-name"
+spec:
+  sla:
+    objective_function: "transaction_response_time"
+    sla_class: "response_time"
+    direction: "minimize"
+    function_variables:
+    - name: "transaction_response_time"
+      query: "application_org_acme_microprofile_metrics_PrimeNumberChecker_checksTimer_mean_seconds"
+      datasource: "prometheus"
+      value_type: "double"
+  mode: "show"
+  selector:
+    matchLabel: "app.kubernetes.io/name"
+    matchLabelValue: "petclinic-deployment"
+    matchRoute: ""
+    matchURI: ""
+    matchService: "https"
+  datasource:
+    name: "prometheus"
+    value: "http://192.168.49.2:31609"
```

Comment:
IP and Port no. should not be hardcoded. This should be replaced dynamically when the test is run.